nixos/modules/services/web-apps/plausible.nix

```diff
@@ -180,7 +180,8 @@ in {
       enable = true;
     };
 
-    services.epmd.enable = true;
+    # See note [plausible-needs-no-erlang-distributed-features].
+    services.epmd.enable = false;
```

Comment:
The default for `services.epmd` is false, so just unsetting the option would do what we want, setting it explicitely to `false` might clash with another service using it and breaking some setups.

Reply:
Done.